code-studio/src/css/plc/content_creation

```diff
@@ -0,0 +1,8 @@
+.outlinedSection {
```

Comment:
I think the file name should also be dash case. I'm not sure why the existing .scss in code-studio uses underscores, but we use dash case for all our other css and scss files.